source/Enemy.hx

```diff
@@ -39,8 +39,8 @@ class Enemy extends FlxSprite
 	public override function update(elapsed:Float):Void
 	{		
 		super.update(elapsed);	
-		if (x > 670)
-		x = -30;
+		//if (x > 670)
```

Comment:
I had left this in mostly because I debated whether or not I wanted to have enemies be warped back to their initial spawn after they left the screen like how Mario's enemies do.